editor/filesystem_dock.cpp

```diff
@@ -2368,8 +2378,8 @@ void FileSystemDock::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("_feature_profile_changed"), &FileSystemDock::_feature_profile_changed);
 
+	ADD_SIGNAL(MethodInfo("inherent", PropertyInfo(Variant::STRING, "file")));
```

Comment:
Should still be changed to `inherit` (same for `FILE_INHERENT` and other occurrences).
For the reference: https://en.wiktionary.org/wiki/inherent